Gemfile

```diff
@@ -61,7 +70,5 @@ group :development, :test do
 	gem 'factory_girl_rails', '~> 4.0'
 end
 
-group :production do
-  gem 'pg'
-end
-
+# production database
+gem 'pg', group: :production
```

Comment:
You are right that this is more efficient.

If you ever need to add more gems to your production group you'll need to expand it back out again or have multiple lines with this tag. 

Either way is fine. I probably wouldn't have changed it but if I was writing it up for the first time I might have written it like you did. Generally I don't change what exists unless I feel that the change will have a long term benefit of being better. Since this might need to revert, I might have left it as it is.